src/NzbDrone.Core/Notifications/Ntfy/NtfyProxy.cs

```diff
@@ -153,5 +154,10 @@ private void SendNotification(string title, string message, HttpRequestBuilder r
                 throw new NtfyException("Unable to send text message: {0}", ex, ex.Message);
             }
         }
+
+        private string GetBase64EncodedValue(string input)
+        {
+            return $"=?UTF-8?B?{input.EncodeBase64()}?=";
+        }
```

Comment:
I can't see where this is being used?

Reply:
I must've not saved before committing it, but this was ntfy.sh's suggested way to include non-ascii in the headers, but it wasn't working in my testing and query string params are just nicer.